src/App.tsx

```diff
@@ -19,42 +19,34 @@
  * SPDX-License-Identifier: Apache-2.0
  ********************************************************************************/
 
```

Comment:
Please fix [License header](https://github.com/catenax-ng/foss-example/blob/main/eclipse-tractusx-repositories/CopyrightAndLicenseHeader_2.txt#L3), L4

Please do this for all files, where you have added a license header! I won't add several comments for the same finding.